{% extends "base.html" %} {% block title %}Experience{% endblock %} 

{% block content%}
<div class="mx-auto" >
    <div class="container pl-30 pr-30" id="header">
        <div class="col text-center">
        
            <div class="p-3 "></div>
            <h1 class="text-center">Qualifications</h1>
            <h6 class="text-left"> MSC BIOINFORMATICS, QUEEN MARY UNIVERSITY, LONDON, SEPTEMBER 2021 – SEPTEMBER 2022 </h6>
            <h6 class="text-left"> Awarded: 2:1 </h6>
            <p class="text-center" > 
            Modules: Statistics for Bioinformatics (62%), Coding for scientists (57%), Genome Bioinformatics (72%), Post-Genomics (70%) and Software Development Group Project (75%). </p>
            <h6 class="text-left">  BSC (HONS) BIOLOGICAL SCIENCE, UNIVERSITY OF BEDFORDSHIRE, ENGLAND, OCTOBER 2017- JUNE 2020 </h6>
            <h6 class="text-left">  Awarded: 2:1 </h6>

            <h1 class="text-center">Experience</h1>
            <h6 class="text-left">Research Assistant, Wellcome Sanger Institute, Saffron Walden  November 2020 – July 2021 </h6>
            <ul class="list-unstyled">
                <ul>
              The ability to solve problems and communicate with researchers was gained as a Research Assistant within a NGS laboratory.
              One of the day-to-day activities included processing plates, whereby each plate contained 382 samples.
              A colleague required help with an error message.
              The error occurred in response to pipettes being absent which is described in the error message - “probe on position (1) = 0, (2) = 0, (3) = 0”. 
              This prevented plates from being processed. 
              Explained what the error stated to the team member and mentioned replacing the tip box with a fresh full tip box and then selecting retry. 	
              As a result, the programme resumed, the plate was processed, and the issue was logged.
              QPCR data analysis was performed on 382 samples for each plate during the quality control stage. 
                </ul>
            </ul>
        
        
            <h6 class="text-left" >  Outreach Associate, University of Bedfordshire, Luton 
                <brk> November 2019 - April 2020 </h6>
            <ul class="list-unstyled">
                <ul>
             Presentation skills were demonstrated during outreach activities.
            Activities were planned within a team of two.
            One of the activities involved explaining different scientific techniques currently being utilised in the industry. For example, a Microsoft PowerPoint assisted with explaining how CRISPR technology genetically modifies organisms.
            Consequently, the children and their teachers gave positive feedback and asked questions at the end of the session.
                </ul>
            </ul>
                <h1 class="text-center">Skills</h1>
                <ul class="list-unstyled"></ul>
                    <ul> Python, Biopython, R, data structures, algorithms, DNA barcoding, Microsoft Excel, HPC, JSON, API, TCGA, Ensembl, VEP, cBioportal, COSMIC, Uniprot, NCBI databases, Fastqc, Cutadapt, IGV, Snakemake, GATK, Protein data bank, Kegg, Pymol, Reactome, String, Cytoscape, Bcftools, Samtools, Pandas, Bash, Linear models, Hypothesis testing, Bayesian statistics, SVM and PCA.  </ul>   

            
        </div>
        
    </div>
</div>
 
{% endblock %}  

