{% extends "base.html" %} {% block title %}Projects{% endblock %} 

{%block content%}
<div class="container">
    <h1 class="text-left">Projects</h1>
    <div class="row text-center">
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <h6 class="text-left"> A pipeline for identifying variants in zebrafish, April 2022-August 2022</h6>
            <p class="text-left" >  ·	A Snakemake pipeline which implemented singularity containers, python/shell scripts and YAML was developed. 
                ·	The singularity containers packaged 3 variant-calling tools (e.g., GATK), an aligner and other tools to identify variants in the zebrafish (model) from exome sequence alignments in BAM format.
                ·	Natural mutations were found using a python script and annotated.
                ·	Python wrangled variants to obtain the correct format for statistical analysis.
                ·	Natural variants from several hundred samples are to be deposited in the European Variation Archive. 
            </p>
        </div>
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <div class="embed-responsive embed-responsive-16by9">
                <iframe class="embed-responsive-item" src="../static/video/variant_calling.mp4" allowfullscreen></iframe>
              </div> 
            </div>
        </div>
    </div>
</div>
<div class="container">
    <div class="row text-center">
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <h6 class="text-left"> Integrating transcriptomics and proteomics data from ovarian cancer samples to study tumour microenvironment, December 2021</h6>
            <p class="text-left" >  ·	Experience with R for the analysis of transcriptomic data was highlighted by studying the tumour microenvironment.
                ·	The difference in gene expression of a biomarker for normal and cancer samples was explored. Also, the correlation between transcript and protein levels of the biomarker was visualised.
                ·	Also, genes that have a transcript that significantly correlates with protein levels and genes with significant differential gene expression between cancer and normal samples were identified.
                ·	A hierarchical cluster analysis was performed using transcript expression data.
            </p>
        </div>
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <div class="embed-responsive embed-responsive-16by9">
                <iframe class="embed-responsive-item" src="../static/video/Data_integration.mp4" allowfullscreen></iframe>
              </div> 
            </div>
        </div>
    </div>
</div>
<div class="container">
    <div class="row text-center">
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <h6 class="text-left"> Evaluate a machine learning tool to diagnose Crohn’s disease using metabolomic data, November 2021</h6>
            <p class="text-left" >  ·	Experience with omics analysis was gained from testing and training a machine learning model.
                ·	The trends of total ion chromatography for each sample with 50 observations were explored.
                ·	Also, scaled metabolomic data from 23 samples were pre-processed using PCA.
                ·	The pre-processed metabolomic patient data was utilised for training and testing an ensemble of SVM classifiers.
                ·	3 metrics were used to evaluate the reliability of models generated by different bootstrap and ensemble numbers.
            </p>
        </div>
        <div class="col-6 col-md-6">
            <div class="p-2 "></div>
            <div class="embed-responsive embed-responsive-16by9">
                <iframe class="embed-responsive-item" src="../static/video/ml_project.mp4" allowfullscreen></iframe>
              </div> 
            </div>
        </div>
    </div>
</div>
{%endblock%}